backend/package.json

```diff
@@ -18,10 +18,17 @@
     "star-tabulator": "^1.0.2",
     "@types/express": "^4.17.13",
     "@types/node": "^16.11.10",
+    "axios": "^0.24.0",
+    "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
+    "node-fetch": "^3.1.0",
+    "qs": "^6.10.2",
     "ts-node": "^10.4.0",
     "typescript": "^4.5.2"
   },
   "devDependencies": {
+    "@types/cookie-parser": "^1.4.2",
```

Comment:
cors and cookie-parser were put in devDependencies which is ignored by heroku. Move to dependencies instead